### PR TITLE
Wearing freshly washed clothes makes you feel better

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
add: Wearing a jumpsuit that's been washed in a washing machine
will give you a positive moodlet.
/:cl:

In addition, cleaned up the washing machine code so there no longer
needs to be a "has_corgi" var.

Something with dodgy colours no longer clobbers the examine text on a jumpsuit

### CODE REVIEW

**Comment by JJRcop** [comment]
What do you think about changing `freshly_laundered` into a timestamp that it was laundered, and you only get the moodlet if it was within some time (long? short?)

**Comment by optimumtact** [comment]
@JJRcop that does imply either the moodlet or the item has to check periodically though

**Comment by coiax** [comment]
@Tlaltecuhtli Good idea, but out of scope.

File: code/modules/clothing/under/_under.dm [hunk 1]
```diff
@@ -44,15 +46,15 @@
...
+	. = ..()
 	if(random_sensor)
 		//make the sensor mode favor higher levels, except coords.
 		sensor_mode = pick(SENSOR_OFF, SENSOR_LIVING, SENSOR_LIVING, SENSOR_VITALS, SENSOR_VITALS, SENSOR_VITALS, SENSOR_COORDS, SENSOR_COORDS)
-	adjusted = NORMAL_STYLE
-	..()
 
 /obj/item/clothing/under/equipped(mob/user, slot)
 	..()
+	to_chat(world, "<span class='reallybig'>[user], [slot]</span>")
```
Comment (Cyberboss) [inline code/modules/clothing/under/_under.dm]: .

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Pull request merged**